src/components/editor/faultTree/menu/faultEvent/FaultEventMenu.tsx

```diff
@@ -68,27 +69,38 @@ const FaultEventMenu = ({ shapeToolData, onEventUpdated, refreshTree, rootIri }:
       setAtaSystem(undefined);
     }
 
-    if (filteredPartNumber.length === 1 && filteredPartNumber[0].partNumber && filteredPartNumber[0].stock) {
+    if (filteredPartNumber.length === 1 && filteredPartNumber[0].partNumber) {
       setPartNumber(filteredPartNumber[0].partNumber);
-      setStock(filteredPartNumber[0].stock);
     } else {
       setPartNumber(undefined);
-      setStock(undefined);
     }
 
     if (shapeToolData?.supertypes?.behavior?.item?.quantity) {
       setQuantity(shapeToolData?.supertypes?.behavior?.item?.quantity);
     } else {
       setQuantity(undefined);
     }
+
+    if (shapeToolData?.supertypes?.behavior?.item?.stock) {
+      setStock(shapeToolData?.supertypes?.behavior?.item?.stock);
+    } else {
+      setStock(undefined);
+    }
+
+    if (shapeToolData?.supertypes?.behavior?.item?.schematicDescription) {
+      setSchematicDescription(shapeToolData?.supertypes?.behavior?.item?.schematicDescription);
+    } else {
+      setSchematicDescription(undefined);
+    }
   }, [shapeToolData]);
 
   const basedFailureRate = shapeToolData?.supertypes?.supertypes?.hasFailureRate?.estimate?.value;
   const requiredFailureRate = shapeToolData?.supertypes?.hasFailureRate?.requirement?.upperBound;
 
   return (
     <Box paddingLeft={2} marginRight={2}>
-      <FaultEventShapeToolPane data={shapeToolData} onEventUpdated={onEventUpdated} refreshTree={refreshTree} />
+      {/* Hidden to prevent app crush after root node select (Bug with UI). Will be fixed in next PR */}
```

Comment:
- put it to separate PR @Kasmadei

Reply:
Removed from this PR

Reply:
@Kasmadei i did not see any issue for that so i created it, i thought you will create PR with this commit as temporal workaround for the issue